src/Controller/Organization/PackageController.php

```diff
@@ -46,35 +46,40 @@ public function packageNew(Organization $organization, Request $request, GithubA
         $form->get('type')->setData($type);
         $response = null;
 
-        switch ($type) {
-            case null:
-                $form->remove('Add');
-                break;
-            case 'git':
-            case 'mercurial':
-            case 'subversion':
-            case 'pear':
-                $response = $this->packageNewFromUrl('url', $form, $organization, $request);
-                break;
-            case 'path':
-            case 'artifact':
-                $response = $this->packageNewFromUrl($type, $form, $organization, $request);
-                break;
-            case 'github':
-                $response = $this->packageNewFromGitHub($form, $organization, $request, $githubApi);
-                break;
-            case 'gitlab':
-                $response = $this->packageNewFromGitLab($form, $organization, $request, $gitlabApi);
-                break;
-            case 'bitbucket':
-                $response = $this->packageNewFromBitbucket($form, $organization, $request, $bitbucketApi);
-                break;
-            default:
-                throw new NotFoundHttpException();
-        }
+        try {
+            switch ($type) {
+                case null:
+                    $form->remove('Add');
+                    break;
+                case 'git':
+                case 'mercurial':
+                case 'subversion':
+                case 'pear':
+                    $response = $this->packageNewFromUrl('url', $form, $organization, $request);
+                    break;
+                case 'path':
+                case 'artifact':
+                    $response = $this->packageNewFromUrl($type, $form, $organization, $request);
+                    break;
+                case 'github':
+                    $response = $this->packageNewFromGitHub($form, $organization, $request, $githubApi);
+                    break;
+                case 'gitlab':
+                    $response = $this->packageNewFromGitLab($form, $organization, $request, $gitlabApi);
+                    break;
+                case 'bitbucket':
+                    $response = $this->packageNewFromBitbucket($form, $organization, $request, $bitbucketApi);
+                    break;
+                default:
+                    throw new NotFoundHttpException();
+            }
```

Comment:
this will change (when introduce integrations), but for now is our small technical dept :wink: